src/shared/utils.cc

```diff
@@ -22,6 +22,9 @@
 #include "cleanup.h"
 #include <assert.h>
 #include <fty_common.h>
+#include <mutex>
+
+std::mutex timegm_mux;  // Mutex for my_timegm function which is not thread-safe
```

Comment:
shall be declared static in my_timegm()

Reply:
Thanks you are right it could be, I will changed it if the PR is reopened